### PR TITLE
Linting Tweaks

### PR DESCRIPTION
**Pull request by joebell93**
## Description

- Enforce `eslint-config-sky` across the entire project (allowing ES6-specific rules to be removed)

## Related Issue

#392

## Motivation and Context

Improve dev experience

## How Has This Been Tested?

All tests pass

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.

### CODE REVIEW

**Comment by joebell93** [comment]
Will update Stylelint in `stylelint-config-sky-uk` to fix the VSCode Markdown error:
https://github.com/stylelint/stylelint/issues/3212

File: packages/sky-toolkit-core/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
+/* Disable ES6-Specific Rules */
```
Comment (StefanMcCready) [inline packages/sky-toolkit-core/index.js]: I presume ways of adhering to these rules have been explored before disabling them?

Comment (joebell93) [reply packages/sky-toolkit-core/index.js]: These two index files aren't compiled by Babel, so, therefore, aren't necessary

(imo using Babel for these 2 index files would be overkill)

File: packages/sky-toolkit/index.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
-var version = require('./package.json').version;
+/* Disable ES6-Specific Rules */
+/* eslint-disable no-var, prefer-template */
```
Comment (StefanMcCready) [inline packages/sky-toolkit/index.js]: Same as comment above. I just get a bit uneasy with disabling rules

Comment (joebell93) [reply packages/sky-toolkit/index.js]: https://github.com/sky-uk/toolkit/pull/395#discussion_r178755525

**Pull request merged**